diff --git a/external/Hyena/Hyena.Data.Sqlite/Hyena.Data.Sqlite.csproj b/external/Hyena/Hyena.Data.Sqlite/Hyena.Data.Sqlite.csproj
index cabce10..21ed285 100644
--- a/external/Hyena/Hyena.Data.Sqlite/Hyena.Data.Sqlite.csproj
+++ b/external/Hyena/Hyena.Data.Sqlite/Hyena.Data.Sqlite.csproj
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="3.5">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
diff --git a/external/Hyena/Hyena.Gui/Hyena.Gui.csproj b/external/Hyena/Hyena.Gui/Hyena.Gui.csproj
index 0cf3088..1749a5e 100644
--- a/external/Hyena/Hyena.Gui/Hyena.Gui.csproj
+++ b/external/Hyena/Hyena.Gui/Hyena.Gui.csproj
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="3.5">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
@@ -66,9 +66,7 @@
     <Reference Include="gdk-sharp">
       <SpecificVersion>False</SpecificVersion>
     </Reference>
-    <Reference Include="Mono.Cairo">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..bin\Mono.Cairo.dll</HintPath>
+    <Reference Include="Mono.Cairo, Version=4.0.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756">
     </Reference>
     <Reference Include="pango-sharp">
       <SpecificVersion>False</SpecificVersion>
diff --git a/external/Hyena/Hyena.Gui/Makefile.am b/external/Hyena/Hyena.Gui/Makefile.am
index 74fd6df..4a5696e 100644
--- a/external/Hyena/Hyena.Gui/Makefile.am
+++ b/external/Hyena/Hyena.Gui/Makefile.am
@@ -1,5 +1,5 @@
 ASSEMBLY = Hyena.Gui
-ASSEMBLY_BUILD_FLAGS = -unsafe
+ASSEMBLY_BUILD_FLAGS = -unsafe -sdk:4
 TARGET = library
 LINK = -r:ICSharpCode.SharpZipLib -r:Mono.Posix -r:System -r:System.Core -r:Mono.Cairo \
 	   $(GTKSHARP_LIBS) $(GLIBSHARP_LIBS) \
diff --git a/external/Hyena/Hyena/Hyena.csproj b/external/Hyena/Hyena/Hyena.csproj
index cc18abc..b7014c2 100644
--- a/external/Hyena/Hyena/Hyena.csproj
+++ b/external/Hyena/Hyena/Hyena.csproj
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="3.5">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
diff --git a/external/taglib-sharp/src/taglib-sharp.csproj b/external/taglib-sharp/src/taglib-sharp.csproj
index 98e277a..ebca24a 100644
--- a/external/taglib-sharp/src/taglib-sharp.csproj
+++ b/external/taglib-sharp/src/taglib-sharp.csproj
@@ -11,7 +11,6 @@
     <ApplicationIcon>.</ApplicationIcon>
     <AssemblyName>taglib-sharp</AssemblyName>
     <ReleaseVersion>2.0.4.0</ReleaseVersion>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>True</DebugSymbols>
