diff --git a/external/Hyena/Hyena.Data.Sqlite/Hyena.Data.Sqlite.csproj b/external/Hyena/Hyena.Data.Sqlite/Hyena.Data.Sqlite.csproj
index cabce10..21ed285 100644
--- a/external/Hyena/Hyena.Data.Sqlite/Hyena.Data.Sqlite.csproj
+++ b/external/Hyena/Hyena.Data.Sqlite/Hyena.Data.Sqlite.csproj
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="3.5">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
diff --git a/external/Hyena/Hyena.Gui/Hyena.Gui.csproj b/external/Hyena/Hyena.Gui/Hyena.Gui.csproj
index 0cf3088..539cfcc 100644
--- a/external/Hyena/Hyena.Gui/Hyena.Gui.csproj
+++ b/external/Hyena/Hyena.Gui/Hyena.Gui.csproj
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="3.5">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
diff --git a/external/Hyena/Hyena/Hyena.csproj b/external/Hyena/Hyena/Hyena.csproj
index cc18abc..b7014c2 100644
--- a/external/Hyena/Hyena/Hyena.csproj
+++ b/external/Hyena/Hyena/Hyena.csproj
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="3.5">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
diff --git a/external/taglib-sharp/src/taglib-sharp.csproj b/external/taglib-sharp/src/taglib-sharp.csproj
index 98e277a..ebca24a 100644
--- a/external/taglib-sharp/src/taglib-sharp.csproj
+++ b/external/taglib-sharp/src/taglib-sharp.csproj
@@ -11,7 +11,6 @@
     <ApplicationIcon>.</ApplicationIcon>
     <AssemblyName>taglib-sharp</AssemblyName>
     <ReleaseVersion>2.0.4.0</ReleaseVersion>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>True</DebugSymbols>
@@ -22,11 +21,12 @@
     <WarningLevel>4</WarningLevel>
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
     <CheckForOverflowUnderflow>True</CheckForOverflowUnderflow>
-    <AssemblyOriginatorKeyFile>.</AssemblyOriginatorKeyFile>
+    <AssemblyOriginatorKeyFile>..\taglib-sharp.snk</AssemblyOriginatorKeyFile>
     <Execution>
       <Execution clr-version="Net_2_0" />
     </Execution>
     <DefineConstants>HAVE_SHARPZIPLIB</DefineConstants>
+    <SignAssembly>true</SignAssembly>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>none</DebugType>
@@ -287,9 +287,6 @@
   </ItemGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
   <ItemGroup>
-    <None Include="AssemblyInfo.cs.in" />
-    <None Include="policy.2.0.taglib-sharp.config.in" />
-    <None Include="taglib-sharp.snk" />
     <None Include="TagLib\TagLib.sources" />
   </ItemGroup>
 </Project>
